### PR TITLE
Add column to schema, data to table and queries

### PR DESCRIPTION
**Pull request by JbirdL86**
What's new :question: 

- Add species column to the table
- Add data to the table
- Add queries to SQL table

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals values(default, 'Agumon', '20200203', 0, false, 10.23);
 INSERT INTO animals values(default, 'Gabumon', '20181115', 2, true, 8);
 INSERT INTO animals values(default, 'Pikachu', '20210107', 1, false, 15.04);
 INSERT INTO animals values(default, 'Devimon', '20170512', 5, true, 11);
+
+INSERT INTO animals values(default, 'Squirtle', '20200208', 0, false, 11);
```
Comment (DeliceLydia) [inline data.sql]: - [x] Change Weight_kg to `-11.0`

File: data.sql [hunk 2]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals values(default, 'Agumon', '20200203', 0, false, 10.23);
...
+INSERT INTO animals values(default, 'Plantmon', '20221115', 2, true, 5.7);
```
Comment (DeliceLydia) [inline data.sql]: - [x] Change Weight_kg to `-5.7`

File: data.sql [hunk 3]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals values(default, 'Agumon', '20200203', 0, false, 10.23);
...
+INSERT INTO animals values(default, 'Squirtle', '19930402', 3, false, 12.3);
```
Comment (DeliceLydia) [inline data.sql]: - [x] Change weight_kg to `-12.13`

File: data.sql [hunk 4]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals values(default, 'Agumon', '20200203', 0, false, 10.23);
...
+INSERT INTO animals values(default, 'Angemon', '20050612', 1, true, 45);
```
Comment (DeliceLydia) [inline data.sql]: - [x] Change weight_kg to `-45.0`

File: data.sql [hunk 5]
```diff
@@ -4,3 +4,10 @@ INSERT INTO animals values(default, 'Agumon', '20200203', 0, false, 10.23);
...
+INSERT INTO animals values(default, 'Boarmon', '20050607', 7, true, 20.4);
+INSERT INTO animals values(default, 'Blossom', '19981013', 3, true, 17);
```
Comment (DeliceLydia) [inline data.sql]: - [x] Change weight_kg to `17.0` according to the requirements

File: queries.sql [hunk 6]
```diff
@@ -7,4 +7,34 @@ select date_of_birth from animals where  name = 'Agumon' or name = 'Pikachu';
...
+savepoint sp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO sp1;
+UPDATE animals SET weight_kg = weight_kg * -1 where weight_kg < 0;
+COMMIT;
+
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts < 1;
+SELECT avg(weight_kg) FROM animals;
+SELECT neutered, sum(escape_attempts) FROM animals GROUP BY neutered ORDER BY sum DESC;
```
Comment (DeliceLydia) [inline queries.sql]: - [x] Who escapes the most, neutered or not neutered animals? you can do like
you can use `AVG` or `MAX`

File: queries.sql [hunk 7]
```diff
@@ -7,4 +7,34 @@ select date_of_birth from animals where  name = 'Agumon' or name = 'Pikachu';
...
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '20220101';
+savepoint sp1;
```
Comment (DeliceLydia) [inline queries.sql]: - [x] Change this query from date_of_birth > '20220101' to `date_of_birth >= '2022/01/01`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @JbirdL86, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @JbirdL86  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**